sdk/nodejs/nodeBalancerNode.ts

```diff
@@ -47,7 +47,7 @@ import * as utilities from "./utilities";
  *     stickiness: "http_cookie",
  * });
  * const foonode = new linode.NodeBalancerNode("foonode", {
- *     address: pulumi.all(web.map(v => v.privateIpAddress)).apply(privateIpAddress => `${privateIpAddress.map(v => v)}:80`),
+ *     address: pulumi.interpolate`${web.map(v => v.privateIpAddress)}:80`,
```

Comment:
It seems the doc generator could have come up with something more direct here, but generators will generate, and this will still work.

Reply:
FWIW the need to interpolate here (via any method) could also be achieved by separation of port from address in the underlying Terraform provider.